worker/internal/benthos/transformers/generate_street_address.go

```diff
@@ -20,16 +20,27 @@ type Address struct {
 }
 
 func init() {
-	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length"))
+	spec := bloblang.NewPluginSpec().Param(bloblang.NewInt64Param("max_length")).Param(bloblang.NewInt64Param("seed").Optional())
 
 	err := bloblang.RegisterFunctionV2("generate_street_address", spec, func(args *bloblang.ParsedParams) (bloblang.Function, error) {
 		maxLength, err := args.GetInt64("max_length")
 		if err != nil {
 			return nil, err
 		}
 
+		seed, err := args.GetOptionalInt64("seed")
+		if err != nil {
+			return nil, err
+		}
+		var randomizer *rand.Rand
+		if seed == nil {
+			randomizer = rand.New(rand.NewSource(rand.Int63()))
+		} else {
+			randomizer = rand.New(rand.NewSource(*seed))
+		}
```

Comment:
Separately, we'll probably want to pull this into a func so that other transformers can more easily instantiate a randomizer.

```
seed, err := args.GetOptionalInt64("seed")
if err != nil {
  return nil, err
}
randomizer := getRandomizer(seed)
```

Reply:
yeah im gonna fix the street address one, and then split this out into a separte PR and then come back to it later this week